src/app/view/perspectives/chart/chart-perspective.component.html

```diff
@@ -34,14 +34,14 @@
     <div class="text-center mt-2">
       <attribute-select
         [attributes]="documentAttributes"
-        [emptyDescription]="'X Axis'"
+        [emptyDescription]="'X ' + axisTitle()"
         (attributeSelected)="attributeX = $event">
       </attribute-select>
     </div>
 
   </div>
 </ng-container>
 
-<ng-template #loading>
+<ng-template #loading i18n-title="@@chart.loading">
   Loading...
```

Comment:
I am not sure how this will behave. I would recommend to create inner `<span>` element with `i18n` attribute.